app/javascript/controllers/year_field_controller.js

```diff
@@ -12,8 +12,11 @@ export default class extends Controller {
     if (this.hasChanged)
         return // nothing to do
 
-    if (evt.inputType === "insertReplacementText") // So that typing in digits doesn't do this, only arrow keys or the arrow buttons
-    evt.target.value = evt.target.max
+    // Be careful refactoring this because Firefox and Chrome send different event types when doing arrow buttons
```

Comment:
basically, Firefox sends "insertReplacementText" for arrow keys, and Chrome sends nothing...so instead of checking for an event to trigger the default year, we instead check for typing events to ignore (which can be several different ones)